### PR TITLE
i3, zypper: add Spanish translation

### PR DESCRIPTION
**Pull request by victorhck**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.es/linux/zypper.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# zypper
+
+> Utilidad para la gestión de paquetes en SUSE y openSUSE.
+
```
Comment (CleanMachine1) [inline pages.es/linux/zypper.md:4]: We need a more information link.

Comment (victorhck) [reply pages.es/linux/zypper.md:5]: added link
(should I add the link in zypper English version?)

Comment (navarroaxel) [reply pages.es/linux/zypper.md:5]: @victorhck Yep.

File: pages.es/linux/zypper.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- Instala un nuevo paquete:
+
+`zypper install {{paquete}}`
+
+- Elimina un paquete:
+
+`zypper remove {{paquete}}`
+
+- Actualiza los paquetes instalados con versiones nuevas:
```
Comment (navarroaxel) [inline pages.es/linux/zypper.md:18]:
```suggestion
- Actualiza los paquetes instalados a la última versión disponible:
```

File: pages.es/linux/zypper.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`zypper update`
+
+- Busca en los repositorios un paquete mediante una palabra clave:
+
+`zypper search {{palabra_clave}}`
+
+- Muestra un listado con la información de los repositorios configurados:
+
+`zypper repos -P`
```
Comment (navarroaxel) [inline pages.es/linux/zypper.md:28]: This example is not in the main (English) page.

File: pages.es/linux/zypper.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# zypper
+
+> Utilidad para la gestión de paquetes en SUSE y openSUSE.
+> Más información: <https://en.opensuse.org/SDB:Zypper_manual>
+
+- Actualiza todos los repositorios:
```
Comment (navarroaxel) [inline pages.es/linux/zypper.md:6]:
```suggestion
- Sincroniza la lista de paquetes y versiones disponibles:
```

File: pages.es/linux/zypper.md (hunk 1 above)
Comment (victorhck) [reply pages.es/linux/zypper.md:5]: > @victorhck Yep.

done!

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.es/linux/zypper.md (hunk 3 above)
Comment (victorhck) [reply pages.es/linux/zypper.md:28]: yes, I know. I though that would be ok to add a example about how zypper also can manage repos

File: pages.es/linux/zypper.md (hunk 4 above)
Comment (victorhck) [reply pages.es/linux/zypper.md:6]: Running a zypper --help in my Tumbleweed with that command that's the text you can read. I though that this text would be more accurate.

Comment (navarroaxel) [reply pages.es/linux/zypper.md:6]: I understand but we want a literal translation of the main pages, it's easier to keep them in sync. I think you should split this PR into 2, keep this for edit the main `zypper` page and add the Spanish translation; and another one to add the translation of `i3` and additional pages if you want to add more.

File: pages.es/linux/zypper.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# zypper
+
+> Utilidad para la gestión de paquetes en SUSE y openSUSE.
+> Más información: <https://en.opensuse.org/SDB:Zypper_manual>
```
Comment (CleanMachine1) [inline pages.es/linux/zypper.md:4]:
```suggestion
> Más información: <https://en.opensuse.org/SDB:Zypper_manual>.
```

File: pages/linux/zypper.md [hunk 6]
```diff
@@ -1,6 +1,7 @@
 # zypper
 
 > SUSE & openSUSE package management utility.
+> More information: <https://en.opensuse.org/SDB:Zypper_manual>
```
Comment (marchersimon) [inline pages/linux/zypper.md:4]:
```suggestion
> More information: <https://en.opensuse.org/SDB:Zypper_manual>.
```

File: pages.es/linux/zypper.md (hunk 3 above)
Comment (marchersimon) [reply pages.es/linux/zypper.md:28]: Would it be possible that you could add this example to the English page also?

**Comment by victorhck** [comment]
I'll close this PR. And will split the changes.

**Comment by marchersimon** [comment]
@victorhck you don't have to. We often have PRs for translating multiple pages.

**Comment by victorhck** [comment]
> @victorhck you don't have to. We often have PRs for translating multiple pages.

Well, I have created PR #6225 to modify the zypper page.
After that I will create the i3 and zypper PR for Spanish translation.

I think this is more accurate...

**Pull request closed without merging**